app/code/Magento/Ui/view/base/web/js/modal/prompt.js

```diff
@@ -152,10 +152,9 @@ define([
         /**
          * Close modal window
          */
-        closeModal: function (result) {
+        closeModal: function (event, result) {
```

Comment:
Should the `event` be used somehow in the method body?

Reply:
Yes, You are right and event is used in the method body